en/camera/mavlink_camera.md

```diff
@@ -0,0 +1,88 @@
+# MAVLink Cameras
```

Comment:
@julianoes Can you sanity check this document and the one above it.

This attempts to capture what you do for a MAVLink camera more clearly. I think it is much better, but I am biased :-) The thing is, with this you can just keep on adding more links to other camera managers and so on - its a scalable doc. 

Also please confirm you are OK with the way we refer to your camera manager. I have tried to capture that it is "community" not something you will necessarily fix, and that it is a good example of MAVSDK. It would be great to have a guide sometime on how you constructed it.